src/constants.rs

```diff
@@ -69,7 +69,7 @@ pub fn classify(hrp: &str) -> Option<Network> {
     match hrp {
         "bc" => Some(Network::Bitcoin),
         "tb" => Some(Network::Testnet),
-        "sb" => Some(Network::Signet),
+        "tb" => Some(Network::Signet),

```

Comment:
This looks like a non-1:1 mapping, resulting in duplicate entries in the `match` block.

Reply:
Ah, yeah. Signet and testnet both use 'tb' -- removed Signet from this list.